aviary/subsystems/geometry/test/test_gasp_geom_builder.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+import numpy as np
+import openmdao.api as om
+
+from openmdao.utils.assert_utils import assert_check_partials, assert_near_equal
+
+from aviary.subsystems.geometry.geometry_builder import CoreGeometryBuilder
+from aviary.variable_info.enums import LegacyCode
+from aviary.variable_info.variables import Aircraft
+from aviary.variable_info.variable_meta_data import _MetaData as BaseMetaData
+import aviary.api as av
+
+GASP = LegacyCode.GASP
+
+
+class TestGASPGeomBuilder(av.TestSubsystemBuilderBase):
+    """
+    That class inherits from TestSubsystemBuilder. So all the test functions are
+    within that inherited class. The setUp() method prepares the class and is run
+    before the test methods; then the test methods are run.
+    """
+
+    def setUp(self):
+        self.subsystem_builder = CoreGeometryBuilder(
+            'core_geometry',
+            BaseMetaData,
+            use_both_geometries=True,
```

Comment:
Should this be false?

Reply:
Changed to use_both_geometries=False. Should we test both?